src/lib/components/ScheduleWidget/ScheduleWidgetCalendar.svelte

```diff
@@ -172,26 +176,38 @@
 					{event.title}
 					{#if event.endsAt && isSameDay(event.endsAt, event.beginsAt)}
 						{t('schedule.dateText', {
-							beginTime: event.beginsAt.toLocaleTimeString($locale, {
+							beginTime: toZonedTime(
+								event.beginsAt,
+								schedule.timezone ?? Intl.DateTimeFormat().resolvedOptions().timeZone
```

Comment:
The expression `schedule.timezone ?? Intl.DateTimeFormat().resolvedOptions().timeZone` is repeated multiple times, it's long and clutters the code.

We could probably extract the pattern `toZonedTime(_, schedule.timezone ?? Intl.DateTimeFormat().resolvedOptions().timeZone)` into a function; this would make the intent clear and improve the readability.

I won't comment other instances, but they should all be addressed.